pkg/controller/plan/adapter/ocp/validator.go

```diff
@@ -158,24 +160,27 @@ func (r *Validator) NetworksMapped(vmRef ref.Ref) (ok bool, err error) {
 					"vm",
 					vmRef.String(),
 				)
+				r.log.Error(err, "Pod network not found.")
+
 				return false, err
 			}
 		} else if net.Multus != nil {
-			namespace, name := getNetworkNameAndNamespace(net.Multus.NetworkName, &vmRef)
-
+			name, namespace := getNetworkNameAndNamespace(net.Multus.NetworkName, &vmRef)
 			_, found := r.plan.Referenced.Map.Network.FindNetworkByNameAndNamespace(namespace, name)
 			if !found {
 				err = liberr.Wrap(
 					err,
 					"Multus network not found.",
-					"vm",
+					"network",
 					fmt.Sprintf("%s/%s", namespace, name),
 				)
+				r.log.Error(err, "Multus network not found.")
```

Comment:
does it include the name of the network?

Reply:
no, but do we want to copy the entire thing twice?

Reply:
ok, let's leave it for now